benchmarks/crossvalidation.py

```diff
@@ -0,0 +1,26 @@
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import GridSearchCV, cross_val_score
+
+from .common import Benchmark
+from .datasets import _synth_classification_dataset
+
+
+class CrossValidation_bench(Benchmark):
+    def setup(self, params):
```

Comment:
I'd rather split cross_val_score and GridSearchCV. Everywhere else I named the file same as the corresponding scikit-learn module. Here it would be 'model_selection.py'. Then I made a separate class for each class or function we want to benchmark.

It also has the advantage to easily run the benchmark for one class: `asv run -b TheClassName`